sigopt_sklearn/search.py

```diff
@@ -321,34 +325,36 @@ def _fit(self, X, y, parameter_iterable=None):
                                             self.fit_params,
                                             return_parameters=True,
                                             error_score=self.error_score)
-                        for parameters in parameter_configs
-                        for train, test in cv)
+                        for parameters, train, test in jobs)
             except TimeoutError:
                  obs_timed_out = True
 
             if not obs_timed_out:
                 # grab scores from results
-                n_folds = len(cv)
                 for sidx, suggestion in enumerate(suggestions):
-                    out_sidx = n_folds * sidx
-                    scores = [o[0] for o in out[out_sidx:(out_sidx+n_folds)]]
+                    score = out[sidx][0]
                     self.sigopt_connection.experiments(self.experiment.id).observations().create(
                         suggestion=suggestion.id,
-                        value=numpy.mean(scores),
-                        value_stddev=numpy.std(scores))
+                        value=score)
             else:
                 # obsevation timed out so report a failure
                 self.sigopt_connection.experiments(self.experiment.id).observations().create(
                     suggestion=suggestion.id,
                     failed=True)
 
-        # return best SigOpt observation so far
-        best_obs = self.sigopt_connection.experiments(self.experiment.id).fetch().progress.best_observation
-        self.best_params_ = best_obs.assignments.to_json()
+        # return best SigOpt assignments so far
+        best_assignments = self.sigopt_connection.experiments(self.experiment.id).best_assignments().fetch().data
```

Comment:
we probably need to update requirements.txt/setup.py to make sure we have the appropriate version of sigopt for this